task1.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head><title>Document</title>
+	<link rel="stylesheet" type="text/css" href="style.css"></head>
+    <body>
+		<figure>
+			<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRo6dH7z64_o5M1mmtog_c_-6d501ahARPfbcLnn_CTwpsvRGIrvw">
+			<br />
+			<figcaption>javascript</figcaption>
+		</figure>
+		<figure>
+			 <img src="https://spzone-simpleprogrammer.netdna-ssl.com/wp-content/uploads/2016/11/C-1024x576.png">
+			 <br />
+			<figcaption>c++</figcaption>
+		</figure> 
+	    <figure>
+			 <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRWb8CaWP6l7IbnSG93-punTY7PYEBCs6REUjRM3d80h4waQO0Zrw" >
+			 <br />
+			<figcaption>html</figcaption>
+	    </figure>
```

Comment:
I would have embedded these figure tags in ul>li tags.  Because it is a **list** of images with caption.